packages/guides-restructured-text/src/RestructuredText/Directives/Directive.php

```diff
@@ -111,6 +113,21 @@ public function processAction(
     ): void {
     }
 
+    /**
+     * Post processes a node created by the directive to apply common options
+     *
+     * @param DirectiveOption[] $options
+     */
+    final protected function postProcessNode(Node $node, array $options): Node
```

Comment:
we could centralize this by moving this logic to the DirectiveRule in the parser, which would allow you to apply options on all directives. 
See: https://github.com/phpDocumentor/guides/blob/main/packages/guides-restructured-text/src/RestructuredText/Parser/Productions/DirectiveRule.php#L107

Reply:
Good idea!